packages/cli/src/commands/org/config/pull.ts

```diff
@@ -17,20 +18,26 @@ export default class Pull extends Command {
   public static flags = {
     target: flags.string({
       char: 't',
-      description:
-        'The unique identifier of the organization from which to pull the resources. If not specified, the organization you are connected to will be used.',
       helpValue: 'destinationorganizationg7dg3gd',
       required: false,
+      description:
+        'The unique identifier of the organization from which to pull the resources. If not specified, the organization you are connected to will be used.',
     }),
-    // TODO: CDX-447: add flag to select resource types to export
+    resourceTypes: flags.string({
+      char: 'r',
+      helpValue: 'type1 type2',
```

Comment:
Did you test the space separator with an actual run (manual test)? I think this might cause issue (gut feeling)

Reply:
Yes,
I just pulled my entire test org with the command

```
org:config:pull -r extension featuredResult field filter mapping mlModel mlModelAssociation queryParameter queryPipeline queryPipelineCondition rankingExpression  rankingWeight searchPage securityProvider source stopWord subscription thesaurus trigger
```